tests/scripts/csiaddons.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env -S bash +e
```

Comment:
```suggestion
#!/usr/bin/env -S bash
```

whye `+e`?

Reply:
it is used to enable `errexit` option?
Saw it in other scripts too.

Reply:
why not use it with `set +e`?

Reply:
actually I didn't know,`+e` is by default.
I have removed that now.